cmd/kubernetes/operator/kodata/webhook/webhook.yaml

```diff
@@ -86,8 +86,8 @@ metadata:
   name: tekton-operator-proxy-webhook
   namespace: tekton-pipelines
   labels:
-    version: "devel"
-    operator.tekton.dev/release: "devel"
+    version: v0.55.1
```

Comment:
just had a question, so with every release we run the script and get the version replace in all these the files right ?

Reply:
or is this for only release branch ?

Reply:
I see, this is only for release branch 😄